docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '2.2'
+services:
+  rally-tests:
+    build:
+      context: .
+      args:
+        - NEW_USER_UID=${UID}
+        - NEW_USER=${USER}
+    container_name: rally-tests
+    volumes:
+      - $HOME/.gitconfig:/home/${USER}/.gitconfig
+      - $HOME/.github:/home/${USER}/.github
+      - $HOME/.gnupg:/home/${USER}/.gnupg
```

Comment:
Is it required to mount these directories?

Reply:
No, this was left over with the same idea as before, i.e. maybe we could use this for another attempt at #484 in the future. I'll remove it for now.